### DISCUSSION TITLE
Starting with fre:ac

### DISCUSSION THREAD

**Discussion by rmlazzari** [enzo1982/freac#440] [Help & Support] [answered]
Hi, I'm new to fre:ac... actually I'm new to Linux Mint, too. Maybe my question is easy, basic... If so, sorry about that.

I have folders with lots of music encoded as FLAC format. It is well encoded, without clipping, with good volume and so.

But now I'd like to convert these files to MP3 format, the unique format that can be read by some old device, even losing quality. I've started this conversion using Audacity, one by one, but... gee: it's a lot of files! So I'd like to try to convert all the files with a click, all fles in the same folder, with fre:ac 'cause somebody told me this app can do that. My questions:

fre:ac really can do that?
Can I set the app to convert all the files in a folder into MP3, say... 320kbps constant / 48000Hz sample rate?
As I'm not familiar with prompt commands, can I do that using graphic interface?
Could you help me to do that?

Again, please excuse if this is basic questions. And excuse if my English is not too good, I'm a brazilian 65 y.o. retired guy.

Thanks a lot!

P.S.: I'm using Linux Mint Cinnamon and didn't install fre:ac, I've just copied freac-1.1.6-linux-x86_64.AppImage to a folder in the desktop, set the attributes right and... tha's all I did. If you need to know other configuration of my PC, please ask me. Thanks.

**Comment by enzo1982** [comment, marked as answer]
Hi and welcome to the fre:ac community!

Yes, fre:ac can do all you intend to do and you can do it all in the graphical UI.

Did you start up fre:ac yet? If not, you should just need to double click the AppImage file if you already set the attributes correctly.

After that, the basic usage is to drag & drop the files/folders you would like to convert onto fre:ac's joblist, wait until the files are loaded and then click the _Start encoding_ button. You can also select _Encode->Start encoding_ from the menu instead.

In order to create MP3 files, you should select the _LAME MP3 Encoder_ from the drop-down list at the bottom-right of the main window. You can then configure the MP3 encoder settings by clicking the yellow icon next to the drop-down.

To create 320 kbps MP3 files, you should select _Custom settings_ for the _Preset_ option, then choose _CBR (no VBR)_ as the _VBR mode_. This will enable a bitrate slider to the right of the _VBR mode_ box which you can set to 320 kbps.

I would recommend leaving the sample rate unchanged. 48 kHz is the maximum sample rate supported by the MP3 format, so if you have any files with a higher sample rate, those will be converted to 48 kHz automatically. If, however, you have files at 44.1 kHz or lower, you should not resample those to 48 kHz. You will get better results just leaving the sample rate untouched.

One more option that you might want to adjust is the output filename pattern. For fre:ac 1.1.6 you'll find that option right on the first page of the configuration dialog (on the _Encoders_ page). In recent continuous builds, this option was moved to the _Output files_ page.

The output filename pattern determines how fre:ac will generate the names of the converted files. By default it will create a folder for each album and put the individual tracks inside. Some users prefer to keep the original file names, though. To do that, you can set the filename pattern to `<filename>`. You can also let fre:ac preserve the complete original structure of your music library including folder and file names. To do that, set the filename pattern to `<directory>/<filename>`. There are some additional options for that which are explained in more detail [here](https://freac.org/manual/en/howto.html#anchor_pat).

**Reply by rmlazzari** [reply]
Thank you, enzo1982, for the answer. I did what you told me to do - double-click the app file - and it work very fine. Then I set as you recommended and everything run perfectly. A good surprise: it's very easy to rip old CDs. Another good suprise is it is in brazilian portuguese!

Yesterday night I've leaned how to create an icon in the desktop - called "laucher" in Linux - and... what more could I ask? After all fre:ac is a very nice application, easy to install, easy to use and really do what I expect to.

Thank you again!
